### PR TITLE
Change wording from "italicizing" to "emphasizing".

### LINKED ISSUE #7939: Emphasis, Not italicize
Challenge [Change Text Inside an Element Using jQuery](https://www.freecodecamp.com/challenges/change-text-inside-an-element-using-jquery) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; rv:45.0) Gecko/20100101 Firefox/45.0</code>.

The challenge asks for italicizing of the test; however, the test can't be passed unless the emphasis element is used. 

My code:
```html
<script>
  $(document).ready(function() {
    $("#target1").css("color", "red");
    $("#target4").html("<i>#target4</i>");
    
  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

```

### PR DESCRIPTION
**Pull request by ghost**
Adjust wording on "Change Text Inside an Element Using jQuery" to better communicate italicizing by using emphasis element. This closes #7939.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -549,7 +549,7 @@
         "Here's how you would rewrite and italicize the text of our heading:",
```
Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/jquery.json]: There is a second instance of the word italicize here.

You can update your commit by doing `git commit --amend`.

Comment (josectello) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Ah I thought we were intentionally leaving that in to show that we italicizes through the use of emphasis.
Will update later tonight once I get back home!

**Comment by erictleung** [comment]
@josectello Also, I didn't catch in your last PR but can you edit you commit message on GitHub to have `Closes #7939` to reference the issue you're fixing? Thanks!

**Comment by josectello** [comment]
Will do!

**Comment by josectello** [comment]
I'll get on that tonight.

**Comment by josectello** [comment]
Updated the PR with the changes to the file and an amended commit message.

**Comment by erictleung** [comment]
@josectello Git messages themselves don't actually close issues, to my knowledge. They actually need to be in the message you typed in GitHub itself just above.

**Comment by josectello** [comment]
Oh wow, of course. Haha, thanks.

This closes #7939.

**Comment by erictleung** [comment]
@josectello sorry for being nit picky but I meant to put the `Closes` statement in the very first message above you made. You can click on the pencil icon and edit your initial message above.

**Comment by josectello** [comment]
Edited.

**Comment by erictleung** [comment]
@josectello thanks! LGTM :+1:

**Pull request merged**